pkg/services/ngalert/api/lotex_am.go

```diff
@@ -192,3 +192,19 @@ func (am *LotexAM) RoutePostAMAlerts(ctx *models.ReqContext, alerts apimodels.Po
 		nil,
 	)
 }
+
+func (am *LotexAM) RoutePostTestReceivers(ctx *models.ReqContext, config apimodels.TestReceiversConfig) response.Response {
+	yml, err := yaml.Marshal(config)
+	if err != nil {
+		return ErrResp(500, err, "Failed marshal config")
+	}
+
+	return am.withReq(
+		ctx,
+		http.MethodPost,
+		withPath(*ctx.Req.URL, "/api/v1/receivers/test"),
+		bytes.NewBuffer(yml),
+		messageExtractor,
+		nil,
+	)
+}
```

Comment:
This will eventually forward the request to the Alertmanager data source (with ID the request recipient path parameter). But I don't think it supports that endpoint.
I think that we need to just return `http.StatusNotImplemented` in that case
so that this endpoint will only test receivers of the grafana managed Alertmanager instances.